server/auth-router.js

```diff
@@ -165,8 +165,8 @@ module.exports = function authRouter(config = {}) {
 
 			// Handle errors
 			if (req.query.error && !silentAuth) {
-				// Re-attempt login with the login form forced to display if unauthorized error happened
-				if (req.query.error === 'unauthorized') {
+				// Re-attempt login with the login form forced to display if unauthorized/access_denied error happened
+				if (req.query.error === 'unauthorized' || req.query.error === 'access_denied') {
```

Comment:
I don't think we want to always reattempt login when there is an access_denied. There may be some cases where we do want to go to the error page

Reply:
Yeah, I found that we still want to show the error page when the error description contains force_password_reset, and there are probably others. I think this should parse the error description and only return to login in the specific cases we expect, and otherwise show the error page.

Reply:
I guess we did return the unauthorized error in two cases. Just pushed an update to check specifically for the timeout message.